app/scripts/views/home/sections/economy-partial.html

```diff
@@ -0,0 +1,18 @@
+<div class="content-block">
```

Comment:
These partials are all the same for now, but this is where section specific static content for the homepage will go for each section.